scripts/nm-quick.sh

```diff
@@ -734,8 +737,7 @@ install_netmaker() {
 			wget -qO "$SCRIPT_DIR"/docker-compose.override.yml $COMPOSE_OVERRIDE_URL
 		fi
 		wget -qO "$SCRIPT_DIR"/Caddyfile "$CADDY_URL"
-		wget -qO "$SCRIPT_DIR"/netmaker.env "$BASE_URL/scripts/netmaker.env"
-		ln -fs "$SCRIPT_DIR/netmaker.env" "$SCRIPT_DIR/.env"
+		wget -qO "$SCRIPT_DIR"/netmaker.default.env "$BASE_URL/scripts/netmaker.default.env"
```

Comment:
why not just save this as .env instead of creating symbolic link below

Reply:
User editable config has a human readable name and isnt a hidden file.

1. `netmaker.env` - main config, user editable
2. `netmaker.default.env` - defaults shipped with every release
3. `.env` - consumed by machines

Reply:
disagree, but ok.... but you are saving it as netmaker.default.env instead of netmaker.env so the symlink command below will fail

Reply:
> disagree, but ok....

Any reasons?

> you are saving it as netmaker.default.env instead of netmaker.env so the symlink command below will fail

```
$ ln -s missing foo
$ ll
lrwxrwxrwx 1 root root    7 May 24 10:26 foo -> missing
```